index.js

```diff
@@ -7,15 +7,15 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const slackEventsApi = require('@slack/events-api');
 const { WebClient } = require('@slack/client');
+
 const SlackClient = require('./slackClient');
+const slackEvents = require('./slackEvents');
```

Comment:
I think we can incorporate slackEvents inside index.js normally. I don't think there are any benefits for creating an abstraction around the SlackEvents because it's only injecting a SlackClient and calling procedures on it (which is actually more the purpose of the index.js - main program file). I think it makes more sense for that to be done normally in the index.js file.

However, I do think it is a good idea to make a SlackEvents file (in the future). I envision it a little different though. 

It would be like 
```javascript

modue.exports = {
     'EVENT_NAME': CALLBACK_TO_RUN,
    EVENT_NAME2: CALLBACK_TO_RUN2, etc
}
```
but regardless, the main meat of the program is done in the index.js file